src/components/TheFooter.vue

```diff
@@ -105,12 +105,14 @@ const links = {
     { label: 'Blog', url: 'https://blog.aeternity.com' },
     { label: 'Terms of Service', url: '/terms-of-service' },
     { label: 'Privacy Policy', url: '/privacy-policy' },
+    { label: 'Fill out the Survey', url: 'https://aeternity.com/aescan-feedback-survey' },
```

Comment:
```suggestion
    { label: 'Feedback Survey', url: 'https://aeternity.com/aescan-feedback-survey' },
```

I think from a first look it makes more clear what the survey is about.

Reply:
Yes agree. Fixed